client/my-sites/promote-post-i2/main.tsx

```diff
@@ -323,9 +321,10 @@ export default function PromotedPosts( { tab }: Props ) {
 				<div className="promote-post-i2__top-bar-buttons">
 					<InlineSupportLink
 						supportContext="advertising"
+						supportLink={ localizeUrl( 'https://wordpress.com/support/promote-a-post/' ) }
+						supportPostId={ 213203 }
```

Comment:
I initially thought this was needed because right now in production the link does not open the help center, and it does on this PR, so I assumed that WP Admin didn't have access to the [context links](https://github.com/Automattic/wp-calypso/blob/7ba676615efc425157c6bf729af61005c72a0f44/client/components/inline-support-link/context-links.js).

But after taking a deeper look, and I found out that the culprit was the `showSupportModal={ ! isRunningInWpAdmin }` prop that you remove here. That means that the `supportContext` prop is more than enough, and you can get rid of the `supportLink` and `supportPostId` props.